index.html

```diff
@@ -11,14 +11,29 @@
   <title>Awesome books</title>
```

Comment:
- [ ] **Please add a footer to your website as reflected in the wireframe.**

The wireframe provided in the [project requirement repo](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md#project-requirements) contains a footer that is missing on your website. Please add a footer.  Being able to follow a design guideline is a very important skill for any developer.

Reply:
Thank you so much for the review. I had forgotten the footer element. My bad